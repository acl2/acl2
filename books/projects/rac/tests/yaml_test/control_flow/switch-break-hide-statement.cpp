// RAC begin

int foo(int a) {
  int b = 0;

  switch (a) {
  case 1:
    b = 1;
    break;
    b = 2;
  }

  return b;
}
