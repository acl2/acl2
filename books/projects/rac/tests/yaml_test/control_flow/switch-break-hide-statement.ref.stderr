switch-break-hide-statement.cpp:10 (4-9): Statement after break
 10 |     b = 2;
    |     ^^^^^
note: this statment is hidden by
  9 |     break;
    |     ^^^^^
